.github/workflows/ci.yml

```diff
@@ -38,9 +38,17 @@ jobs:
     - name: Format with isort
       run: |
         isort --check --diff .
-    - name: Check with mypy
+    # - name: Check with mypy
+    #   run: |
+    #     mypy tests/
+    - name: Setup node.js (for pyright)
+      uses: actions/setup-node@v1
+      with:
+        node-version: "12"
+    - name: Run type checking
       run: |
-        mypy tests/
+        npm install -g pyright
+        pyright bolts
```

Comment:
this will not fail the CI, right? Is this just to see if you _can_ run pyright? The pyproject should probably have some [pyright config](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#sample-pyprojecttoml-file) stuff

Reply:
it's failing the CI as we speak. or do you mean something else?